core/jazz_acl/components/casbin.js

```diff
@@ -184,48 +189,130 @@ async function addPolicy(serviceId, policies, enforcer) {
 
 /* Get the permissions for a service given a userId */
 async function getPolicyForServiceUser(serviceId, userId, config) {
-  const result = await getPolicies(serviceId, config);
-  if (result && result.error) {
-    return result;
+  if (userId === config.SERVICE_USER) {
+    let result = attachAdminPolicies([serviceId]);
+    return result
+  } else {
+    const result = await getPolicies(serviceId, config);
+    if (result && result.error) {
+      return result;
+    }
+    let policies = formatPolicies(result);
+    let userPolicies = policies.filter(policy => policy.userId === userId);
+    userPolicies = userPolicies.map(policy => { return { permission: policy.permission, category: policy.category } });
+
+    return [{serviceId: serviceId, policies: userPolicies}];
+  }
+
+}
+
+/* fetch list of serviceIds from dynamodb */
+var scanResult;
+async function scanExecute(dynamodb, scanparams, items_formatted) {
```

Comment:
Do it in another file, since this file handles, casbin related stuffs

Reply:
Updated